packages/strapi-generate-new/files/config/environments/development/request.json

```diff
@@ -1,18 +1,20 @@
 {
   "session": {
     "enabled": false,
-    "client": "redis",
-    "settings": {
-      "host": "localhost",
-      "port": 6379,
-      "database": 10,
-      "password": ""
-    }
+    "client": "cookie",
+    "key": "strapi.sid",
```

Comment:
You should put all the cookie settings into a `settings` object to keep consistency between the configurations whatever the client used.